core/src/components/datetime/test/basic/e2e.ts

```diff
@@ -98,6 +98,41 @@ describe('datetime: selecting a day', () => {
 
     expect(newActiveDay.innerText).toEqual('13');
   });
+
+  const testHighlight = async (page) => {
```

Comment:
Any blockers that prevented writing this test in Playwright?

Reply:
... I just forgot :joy: I'll do that, haha.